mappings/package_eforms_sdk1.10_epo4.0/transformation/mappings/Lot_v1.3-1.11.rml.ttl

```diff
@@ -39,7 +39,7 @@ tedm:MG-GreenProcurement-fulfillsStrategicProcurement-Lot_ND-StrategicProcuremen
             rr:objectMap
                 [
                     # while this field is introduced in 1.9,
-                    # the mapping is applicable since time immemorial
+                    # the mapping is theoreticall applicable for all past versions
```

Comment:
Typo: theoreticall -> theoretically

Reply:
Of course this applies to all similar files, as well as other files where this typo appears.